buildout.cfg

```diff
@@ -1,10 +1,24 @@
 [buildout]
-extends = http://download.zope.org/Zope2/index/2.13.19/versions.cfg
-parts = test
+extensions =
+    mr.developer
+extends = 
+    https://raw.githubusercontent.com/zopefoundation/Zope/master/sources.cfg
+    https://raw.githubusercontent.com/zopefoundation/Zope/master/versions-prod.cfg
 develop = .
+parts = 
+    test
+    tox
+auto-checkout =
+    Zope2
```

Comment:
I would have set it to require >= the next alpha upon release, which is just waiting for that Zope 4.0a5 to appear. 1e21be32d4f06718e3eb226e345d87b497379101 fixes it now.